tests/suites/test_suite_psa_crypto.function

```diff
@@ -8502,10 +8503,17 @@ void derive_input(int alg_arg,
                            expected_statuses[i]);
             }
         } else {
-            TEST_EQUAL(psa_key_derivation_input_bytes(
-                           &operation, steps[i],
-                           inputs[i]->x, inputs[i]->len),
-                       expected_statuses[i]);
+            if (input_types[i] == 0) {
+                TEST_EQUAL(psa_key_derivation_input_bytes(
+                               &operation, steps[i],
+                               inputs[i]->x, inputs[i]->len),
+                           expected_statuses[i]);
+            } else if (input_types[i] == 1) {
+                TEST_EQUAL(psa_key_derivation_input_integer(
+                               &operation, steps[i],
+                               *(inputs[i]->x)),
```

Comment:
Hmm, this means we can only pass low values and makes the syntax for passing them complex ("\x00" etc) with portability questions (is `char` signed or not?). Perhaps we should use `strtoull()` instead.

Reply:
When parsing the data, the input value is parsed as a hex array. So I have created a simple helper function to convert the hex array to `uint64_t`. In the data file we have to give the input value in hex without using `0x`.